criu/include/magic.h

```diff
@@ -100,6 +100,7 @@
 #define BPFMAP_FILE_MAGIC    0x57506142 /* Alapayevsk */
 #define BPFMAP_DATA_MAGIC    0x64324033 /* Arkhangelsk */
 #define APPARMOR_MAGIC	     0x59423047 /* Nikolskoye */
+#define UDP_QUEUE_MAGIC	     0x55803795 /* Balasiha */
```

Comment:
0x55483756

Reply:
Thank you, just a bit curiousity how can you get that number? I search for Balasiha's coordinate and the result is 55.7982° N, 37.9680° E

Reply:
I get them from wikipedia:
https://en.wikipedia.org/wiki/Balashikha

55°49′N 37°58′E is is actually the same with 55.7982° N, 37.9680° E. But I think we use the first format for others.